pkg/config/runtime_test.go

```diff
@@ -86,4 +90,5 @@ func TestNewRuntimeConfig(t *testing.T) {
 	_, err = NewRuntimeConfig(k0sVars)
 	assert.Error(t, err)
 	assert.ErrorIs(t, err, ErrK0sAlreadyRunning)
+	t.Cleanup(func() { require.NoError(t, spec.Cleanup()) })
```

Comment:
I'd move this to line 82, so that it's clear that it's cleaning up the `NewRuntimeConfig` call. Also, in test cleanup functions, there's no need to panic with `require` (IIRC it is even problematic), so better use `assert` instead of `require`.